packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/add.html

```diff
@@ -2,23 +2,128 @@
     ng-form="addServiceIntegration"
     data-on-dismiss="$ctrl.cancel()"
     data-primary-label="{{:: 'pci_databases_service_integration_add_confirm' | translate}}"
-    data-primary-disabled="addServiceIntegration.$invalid"
+    data-primary-disabled="addServiceIntegration.$invalid || $ctrl.existingIntegrationError"
     data-primary-action="$ctrl.addServiceIntegration()"
     data-secondary-action="$ctrl.cancel()"
     data-secondary-label="{{:: 'pci_databases_service_integration_add_cancel' | translate}}"
     data-loading="$ctrl.processing"
-    data-heading="{{:: 'pci_databases_service_integration_add' | translate: {engineName: $ctrl.engineName} }}"
+    data-heading="{{:: 'pci_databases_service_integration_add' | translate}}"
 >
-    <oui-field
-        data-label="{{:: 'pci_databases_service_integration_add_service_label' | translate: {engineName: $ctrl.engineName} }}"
+    <oui-message data-type="warning" ng-if="$ctrl.showNoServiceMessage">
```

Comment:
```suggestion
    <oui-message data-type="warning" data-ng-if="$ctrl.showNoServiceMessage">
```

There are others "data-" that are missing in this file, I'll let you add them :)

Reply:
Added missing `data-`: https://github.com/ovh/manager/pull/7186/commits/083c8a9628024ea1f5ec362602d344d16ee0de4c